### PR TITLE
Adds handling of bitcasts in Z3

### PR DESCRIPTION
**Pull request by surovic**

### CODE REVIEW

**Comment by surovic** [comment]
Relates to #38

**Review by pgoodman — APPROVED** [review]

File: rellic/AST/Z3ConvVisitor.cpp [hunk 1]
```diff
@@ -485,7 +486,9 @@ bool Z3ConvVisitor::VisitParenExpr(clang::ParenExpr *parens) {
     // Parens may affect semantics of C expressions
     case Z3_OP_UNINTERPRETED: {
       auto sort = z_sub.get_sort();
+      // DLOG(INFO) << "SATAN 1";
```
Comment (artemdinaburg) [inline rellic/AST/Z3ConvVisitor.cpp:489]: This looks like it is left over from a frustrating debugging session :).

File: rellic/AST/Z3ConvVisitor.cpp [hunk 2]
```diff
@@ -485,7 +486,9 @@ bool Z3ConvVisitor::VisitParenExpr(clang::ParenExpr *parens) {
...
       auto z_paren = z3_ctx->function("Paren", sort, sort);
+      // DLOG(INFO) << "SATAN 2";
```
Comment (artemdinaburg) [inline rellic/AST/Z3ConvVisitor.cpp:491]: Same as above :)

**Review by artemdinaburg — CHANGES_REQUESTED** [review]
Only request is to remove some debugging comments, then its good to go.

File: rellic/AST/Z3ConvVisitor.cpp (hunk 1 above)
Comment (surovic) [reply rellic/AST/Z3ConvVisitor.cpp]: Done.

File: rellic/AST/Z3ConvVisitor.cpp (hunk 2 above)
Comment (surovic) [reply rellic/AST/Z3ConvVisitor.cpp]: Aand done as well.

**Review by pgoodman — APPROVED** [review]

**Pull request merged**